pest/src/position.rs

```diff
@@ -262,21 +260,23 @@ impl<'i> Position<'i> {
     /// function will return `false` and its `pos` will not be updated.
     #[inline]
     pub(crate) fn skip_until(&mut self, string: &str) -> bool {
```

Comment:
`Position::skip_until` and `ParserState::skip_until` are not used any more, as far as I can tell.

Reply:
I guess the question is whether or not we should keep it as a separate function. Another thing I can do is remove this and rename `skip_until_any` to `skip_until` and call it a day.

Reply:
It might be nice to keep both for people using `pest` without `pest_derive`

Reply:
We could leave them both, sure.